client/spec.yaml

```diff
@@ -231,4 +231,199 @@ paths:
+  /api/logs/v1/{tenant}/loki/api/v1/series:
+    parameters:
+      - $ref: ./parameters/parameters.yaml#/components/parameters/tenant
+      - $ref: ./parameters/parameters.yaml#/components/parameters/seriesMatcher
+      - $ref: ./parameters/parameters.yaml#/components/parameters/startTS
+      - $ref: ./parameters/parameters.yaml#/components/parameters/endTS
+    get:
+      tags:
+        - seriesv1
+      summary: Get series for a tenant that matches certain label set
+      operationId: getLogSeries
+      description: |
+        You can get series for a tenant
+      responses:
+        "2XX":
+          description: series for a tenant
+          content:
+            application/json:
+              schema:
+                $ref: ./responses/responses.yaml#/components/schemas/SeriesResponse
+        "4XX":
+          description: Bad request
+        "5XX":
+          description: Server side error
+  /api/logs/v1/{tenant}/loki/api/v1/query_range:
+    parameters:
+      - $ref: ./parameters/parameters.yaml#/components/parameters/tenant
+      - $ref: ./parameters/parameters.yaml#/components/parameters/query
+      - $ref: ./parameters/parameters.yaml#/components/parameters/startTS
+      - $ref: ./parameters/parameters.yaml#/components/parameters/endTS
+      - $ref: ./parameters/parameters.yaml#/components/parameters/limit
+      - in: query
+        name: step
+        description: Query resolution step width
+        schema:
+          type: string
+    get:
+      tags:
+        - queryv1
+      summary: Evaluate range queries for a tenant
+      operationId: getLogRangeQuery
+      description: |
+        You can evaluate range queries for a tenant
+      responses:
+        "2XX":
+          description: range query result for a tenant
+          content:
+            application/json:
+              schema:
+                $ref: ./responses/responses.yaml#/components/schemas/QueryRangeResponse
+        "4XX":
+          description: Bad request
+        "5XX":
+          description: Server side error
+  /api/logs/v1/{tenant}/loki/api/v1/query:
+    parameters:
+      - $ref: ./parameters/parameters.yaml#/components/parameters/tenant
+      - $ref: ./parameters/parameters.yaml#/components/parameters/query
+      - $ref: ./parameters/parameters.yaml#/components/parameters/limit
+      - in: query
+        name: time
+        description: Evaluation timestamp
+        schema:
+          type: string
+          format: rfc3339 | unix_timestamp
+    get:
+      tags:
+        - queryv1
+      summary: Evaluate instant queries for a tenant
+      operationId: getLogInstantQuery
+      description: |
+        You can evaluate instant queries for a tenant
+      responses:
+        "2XX":
+          description: instance query result for a tenant
+          content:
+            application/json:
+              schema:
+                $ref: ./responses/responses.yaml#/components/schemas/QueryResponse
```

Comment:
Shouldn't this be `InstantQueryResponse` here?

Reply:
It refers to InstantQueryResponse in responses